docs/trace/extending-the-sdk/MyEnrichingProcessor.cs

```diff
@@ -0,0 +1,41 @@
+// <copyright file="MyEnrichingProcessor.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Diagnostics;
+using OpenTelemetry;
+
+internal class MyEnrichingProcessor : BaseProcessor<Activity>
+{
+    public override void OnEnd(Activity activity)
+    {
+        // Enrich activity with additional tags.
+        activity.SetTag("mycustomTag", "myCustomTagValue");
+
+        // Enriching from Baggage.
+        // The below snippet adds every Baggage item.
+        foreach (var baggage in Baggage.GetBaggage())
+        {
+            activity.SetTag(baggage.Key, baggage.Value);
+        }
+
+        // The below snippet adds specific Baggage item.
```

Comment:
nit: Maybe be more specific: 
`// The below snippet adds specific Baggage item as an activity tag.`